parade/ddl.sql

```diff
@@ -0,0 +1,160 @@
+CREATE EXTENSION pgcrypto;
+
+CREATE SCHEMA IF NOT EXISTS parade;
+
+-- (requires extensioon pgcrypto)
```

Comment:
extension

Reply:
Removed that comment, and the schema: we don't need either of them.